mm/slub.c

```diff
@@ -2753,13 +2754,17 @@ static __always_inline void maybe_wipe_obj_freeptr(struct kmem_cache *s,
  * Otherwise we can simply pick the next object from the lockless free list.
  */
 static __always_inline void *slab_alloc_node(struct kmem_cache *s,
-		gfp_t gfpflags, int node, unsigned long addr)
+		gfp_t gfpflags, int node, unsigned long addr, size_t cust_size)
```

Comment:
I assume we somewhere clash with 'size'?

If 'cust_size' is the original size, why not 'orig_size'? Or 'caller_size'?

Reply:
Perhaps not in this function, but clashes are indeed possible.
"orig_size" looks good.